source/features/select-all-notifications-shortcut.tsx

```diff
@@ -1,18 +1,24 @@
-import elementReady from 'element-ready';
+import select from 'select-dom';
 import * as pageDetect from 'github-url-detection';
 
 import features from '.';
-import onElementRemoval from '../helpers/on-element-removal';
 
-async function init(signal: AbortSignal): Promise<void> {
-	const selectAllNotifications = await elementReady('.js-notifications-mark-all-prompt');
-	if (selectAllNotifications) { // Notifications page may be empty
-		selectAllNotifications.dataset.hotkey = 'a';
-		await onElementRemoval(selectAllNotifications, signal); // "Select all" checkbox will be replaced if there's more notifications to load #4199
```

Comment:
@fregante There's also another feature which looks at the "select all" checkbox, and might cause a slowdown too:

https://github.com/refined-github/refined-github/blob/bd48d98f400569c02f0a4ac774b1554dde57e710/source/features/select-notifications.tsx#L160

Reply:
If configured correctly, selector-observer already deals with it the right way, probably